bar_industry/data/product_pricelist.xml

```diff
@@ -2,9 +2,7 @@
 <odoo noupdate="1">
     <record id="product_pricelist_1" model="product.pricelist">
         <field name="name">Default</field>
-        <field name="sequence">10</field>
     </record>
```

Comment:
Just spotted that default pricelist is defined twice, so you can remove this manual definition